sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,11 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self._eval_is_real():
```

Comment:
By the way, shouldn't the method `_eval_is_real()` be always true? Because for a general number `a + ib` (a,b are real), `cosh(a + ib) = exp(a)*cos(b)` which is always real.